ssg/build_yaml.py

```diff
@@ -766,6 +767,22 @@ def _get_product_only_references(self):
                     product_references[gref] = gval
         return product_references
 
+    def make_template_product_specific(self, product):
+        product_suffix = "@{0}".format(product)
+
+        if not self.template:
+            return
+
+        not_specific_vars = self.template.get("vars", dict())
+        specific_vars = self._make_items_product_specific(
+            not_specific_vars, product_suffix, True)
+        self.template["vars"] = specific_vars
+
+        not_specific_backends = self.template.get("backends", dict())
+        specific_backends = self._make_items_product_specific(
+            not_specific_backends, product_suffix, True)
+        self.template["backends"] = specific_backends
```

Comment:
Note to myself: This means that "backends" will be always defined, so i we have some time we can drop the else branch or update the code in https://github.com/ComplianceAsCode/content/blob/master/ssg/templates.py#L152